src/main/java/uk/gov/ons/ssdc/caseprocessor/model/dto/PayloadDTO.java

```diff
@@ -7,13 +7,15 @@
 @Data
 @JsonInclude(Include.NON_NULL)
 public class PayloadDTO {
-  private ResponseDTO response;
+  private ReceiptDTO receipt;
   private RefusalDTO refusal;
-  private CollectionCase collectionCase;
-  private UacDTO uac;
-  private InvalidAddress invalidAddress;
-  private FulfilmentDTO fulfilment;
+  private CaseUpdateDTO caseUpdate;
+  private UacUpdateDTO uacUpdate;
+  private InvalidCase invalidCase;
+  private PrintFulfilmentDTO printFulfilment;
   private TelephoneCaptureDTO telephoneCapture;
```

Comment:
I can't see anything about telephone capture in the event dictionary, was it missed out?

Reply:
It's an internal only event, not something we're accepting from outside systems.